### DISCUSSION TITLE
Ajout d'un espace commentaire dans l'onglet "plastique"

### DISCUSSION THREAD

**Discussion by paulineeyherabide** [betagouv/ma-cantine#1903] [Retours utilisateur·rice]
Une utilisatrice me demande s'il est possible d'ajouter un espace commentaire pour les plastiques (comme pour le gaspillage). Avez-vous eu des retours là-dessus ? Qu'en pensez-vous ?

@gauthierandre @JenniferStephan @valeriemerle @Greta712 @YodBetYod

**Comment by ghost** [comment]
De mon côté, je n'ai pas eu de remarques d'utilisateurs à ce sujet pour l'instant.

**Reply by Greta712** [reply]
Pauline, je viens d'avoir une demande similaire : 
- il n'y a pas de champ commentaires, sauf erreur de notre part, notamment pour la catégorie "produits de qualité et durables" ou "substitution des plastiques", dans lequel nous pourrions expliquer ou détailler nos réponses.

Par exemple, sur les montants d'achats de produits de qualité, nous aurions aimé pouvoir détailler notre réponse avec les produits BBC et MSC, que nous n'incluons pas dans les achats Egalim.
